.github/workflows/cd-pipeline.yaml

```diff
@@ -141,3 +142,43 @@ jobs:
           git config user.name "Bitnami Containers"
           git config user.email "bitnami-bot@vmware.com"
           git add bitnami/index.yaml && git commit -m "${chart}-${chart_version}: Update index.yaml" -s && git push
+
+  # If the CD Pipeline does not succeed we should notify the interested agents
+  slack-notif:
+    runs-on: ubuntu-latest
+    needs: vib-publish
+    if: always()
```

Comment:
Yes, we need for the `slack-notif` job to execute every time, regardless of the execution result of the `vib-publish` one. Without it, it would only run if the previous one succeeded because we are using `needs: vib-publish`. Ref: [GH's doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-not-requiring-successful-dependent-jobs)